setup.py

```diff
@@ -2,16 +2,16 @@
 
 from setuptools import setup
 
-with open(os.path.join("scienceworld", "version.py")) as f:
-    VERSION = f.readlines()[0].split("=")[-1].strip("' \n")
+from scienceworld import scienceworld
```

Comment:
I'm getting the error `AttributeError: module 'scienceworld.scienceworld' has no attribute 'version'` later on, so you might fix it by trying the following:

```suggestion
import scienceworld
```

Reply:
@aphedges Good catch. I'll test some more. It worked for me yesterday, but maybe I messed something up. I probably had `scienceworld` in `PYTHONPATH`.

Reply:
I'm not sure how this would behave if you already have a version of `scienceworld` installed (or residing on your PYTHONPATH).